### PR TITLE
docs(typo): correct typo in documentation

### PR DESCRIPTION
**Pull request by irinaschubert**
resolves DSP-xxx

### CODE REVIEW

File: docs/dsp-tools-create.md [hunk 1]
```diff
@@ -728,7 +728,9 @@ Represents a date. It's a string with the format `calendar:start:end`
 Please note that the DateValue is an extremely flexible data type. It can represent an exact date or a date with a given
 uncertainty, and the date can be given in several calendars (currently the Gregorian and the Julian calendars are supported, with
 the Jewish and Islamic coming soon). Internally, a date is always represented as a start and end date. If start and end date
-match, it's an exact date. A value like "1893" will automatically be expanded to a range from January 1st 1893 to December 31st 1893.
+match, it's an exact date. A value like "1893" will automatically be expanded to a range from January 1st 1893 to December 31st
+
+1893.
```
Comment (BalduinLandolt) [inline docs/dsp-tools-create.md:733]: why the new lines?

File: knora/README.md [hunk 2]
```diff
@@ -1,563 +1,682 @@
...
+    - [Bulkimport()](#bulkimport)
+    - [add_resource()](#add_resource)
+    - [upload()](#upload)
+
 ## Knora
-Knora is the base class which handles all the communication with the Knora backend. In order
-to use Knora, the user has to submit his/her credentials to the Knora backend
+
+Knora is the base class which handles all the communication with the Knora backend. In order to use Knora, the user has to submit
+his/her credentials to the Knora backend
```
Comment (BalduinLandolt) [inline knora/README.md:54]:
```suggestion
Knora is the base class which handles all the communication with the DSP API. In order to use Knora, the user has to submit
his/her credentials to the DSP API
```
Not that it matters, but we were replacing all Knora, right?

Comment (BalduinLandolt) [reply knora/README.md:54]: also `his/her` is very gender binary... one could even put `their` to be a bit more modern

**Review by BalduinLandolt — APPROVED** [review]

File: docs/dsp-tools-create.md (hunk 1 above)
Comment (irinaschubert) [reply docs/dsp-tools-create.md:733]: I have no idea what the automatic reformatting was doing here, I corrected it.

File: knora/README.md (hunk 2 above)
Comment (irinaschubert) [reply knora/README.md]: I didn't go through all the file but now that you mention it, I replaced the Knora reference with DSP or DSP API everywhere in this file. I corrected the his/her, too. Thanks for mentioning that.

**Pull request merged**